app/components/PerusePages/History/History.tsx

```diff
@@ -28,39 +28,73 @@ export class History extends Component<HistoryProps, {}> {
 
     render() {
         const { history, isActiveTab, windowId, addTabEnd } = this.props;
-        let historyList = [];
-        const historyArray = [];
-        // eslint-disable-next-line array-callback-return
-        Object.keys( history ).map( function( key ) {
-            historyArray.push( history[key] );
-        } );
-        historyArray.forEach( ( tab, i ) => {
-            if ( tab.history ) {
-                tab.history.forEach( ( tabsHistory ) => {
-                    const historyItem = tabsHistory;
-                    historyList.push( historyItem );
-                } );
-            }
-        } );
         const ignoreProtocolList = ['safe-auth:'];
         const ignoreList = [
             'about:blank',
             'safe-browser://history',
             'safe-browser://bookmarks'
         ];
-        // TODO: uniq by object props, so will be less harsh once we have title etc.
-        historyList = _.uniq( historyList );
-        historyList = historyList.filter( ( url ) => {
-            const urlObj = parse( url );
-            if (
-                ignoreList.includes( url ) ||
-        ignoreProtocolList.includes( urlObj.protocol )
-            ) {
-                return false;
+        const dates = Object.keys( history );
+        let list = [];
+        const parsedList = [];
+        /* eslint-disable array-callback-return */
```

Comment:
same as above

Reply:
@manavbp also when disabling please try and stick to line only disabled (maybe there's a rule to avoid disabling on a block?)